modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/model/extension/xml/MacroExpansionModuleModel.java

```diff
@@ -312,11 +339,15 @@ private ComponentModel createModuleOperationChain(ComponentModel operationRefMod
 
     operationRefModel.getMetadata().getSourceCode().ifPresent(processorChainBuilder::setSourceCode);
 
+    final Optional<String> implicitConfigRef =
```

Comment:
The var name implicitConfigRef might be confusing here, given that not always an implicit config will be used. It might be changed to a more generic name, i.e. configRef